src/content/learn/editor-setup.md

```diff
@@ -1,62 +1,62 @@
 ---
-title: Editor Setup
+title: એડિટર સેટઅપ
 ---
 
 <Intro>
 
-A properly configured editor can make code clearer to read and faster to write. It can even help you catch bugs as you write them! If this is your first time setting up an editor or you're looking to tune up your current editor, we have a few recommendations.
+યોગ્ય રીતે ગોઠવેલ એડિટર કોડને વાંચવા માટે સ્પષ્ટ અને લખવા માટે ઝડપી બનાવી શકે છે. તે તમને ભૂલો લખતા જ તેને પકડવામાં પણ મદદ કરી શકે છે! જો આ તમે પ્રથમ વખત એડિટર સેટઅપ કરી રહ્યાં છો અથવા તમે તમારા વર્તમાન એડિટર ને ટ્યુન કરવા માટે શોધી રહ્યાં છો, તો અમારી પાસે કેટલીક ભલામણો છે.
```

Comment:
I guess translation of configured doesn't required. It would be more clear if we just put કોન્ફિગર્ડ એડિટર instead of  ગોઠવેલ એડિટર.